<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDeviceEditForm</class>
 <widget class="QWidget" name="CDeviceEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="syncScreenCheckBox">
       <property name="text">
        <string>sync screen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="zoomSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="zoomLabel">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>25%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="goBackButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_back</normaloff>
       <normalon>:/img/ic_back_white</normalon>:/img/ic_back</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="homeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_home</normaloff>
       <normalon>:/img/ic_home_white</normalon>:/img/ic_home</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="menuButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_menu</normaloff>
       <normalon>:/img/ic_menu_white</normalon>:/img/ic_menu</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionscreen_record">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/camera_roll</normaloff>
     <normalon>:/img/camera_roll_white</normalon>:/img/camera_roll</iconset>
   </property>
   <property name="text">
    <string>screen record</string>
   </property>
  </action>
  <action name="actionscreen_shot">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/ic_shot</normaloff>
     <normalon>:/img/ic_shot_white</normalon>:/img/ic_shot</iconset>
   </property>
   <property name="text">
    <string>screen shot</string>
   </property>
  </action>
  <action name="actionvolume_up">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/ic_volume_up</normaloff>
     <normalon>:/img/ic_volume_up_white</normalon>:/img/ic_volume_up</iconset>
   </property>
   <property name="text">
    <string>volume up</string>
   </property>
  </action>
  <action name="actionvolume_down">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/ic_volume_down</normaloff>
     <normalon>:/img/ic_volume_down_white</normalon>:/img/ic_volume_down</iconset>
   </property>
   <property name="text">
    <string>volume down</string>
   </property>
  </action>
  <action name="actionvolume_mute">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/ic_volume_mute</normaloff>
     <normalon>:/img/ic_volume_mute_white</normalon>:/img/ic_volume_mute</iconset>
   </property>
   <property name="text">
    <string>volume mute</string>
   </property>
  </action>
  <action name="actionscreen_on">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/turn_on_screen</normaloff>:/img/turn_on_screen</iconset>
   </property>
   <property name="text">
    <string>screen on</string>
   </property>
  </action>
  <action name="actionscreen_off">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/turn_off_screen</normaloff>:/img/turn_off_screen</iconset>
   </property>
   <property name="text">
    <string>screen off</string>
   </property>
  </action>
  <action name="actionhome">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/ic_home</normaloff>
     <normalon>:/img/ic_home_white</normalon>:/img/ic_home</iconset>
   </property>
   <property name="text">
    <string>home</string>
   </property>
  </action>
  <action name="actionback">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/ic_back</normaloff>
     <normalon>:/img/ic_back_white</normalon>:/img/ic_back</iconset>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </action>
  <action name="actionmenu">
   <property name="icon">
    <iconset resource="canary-img.qrc">
     <normaloff>:/img/ic_menu</normaloff>
     <normalon>:/img/ic_menu_white</normalon>:/img/ic_menu</iconset>
   </property>
   <property name="text">
    <string>menu</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="canary-img.qrc"/>
 </resources>
 <connections/>
</ui>
